rules/macos/persistence_periodic_tasks_file_mdofiy.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or modification of the default configuration for the periodic facility.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Persistence via Periodic Tasks"
+note = "Adversaries may abuse the periodic jobs to execute malicious code or maintain persistence."
```

Comment:
```suggestion

```
We should remove this from the notes field and add it to the description (I made a suggestion).